var/spack/repos/builtin/packages/py-galaxy-tool-util/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PyGalaxyToolUtil(PythonPackage):
+    """The Galaxy tool utilities."""
+
+    homepage = "https://github.com/galaxyproject/galaxy"
+
+    version(
+        "22.1.5",
+        url="https://files.pythonhosted.org/packages/76/b0/4745ccb75fdb5c48a27a73900f8430119afe6c7c95b1aafed004f093788c/galaxy_tool_util-22.1.5-py2.py3-none-any.whl",
+        sha256="062a758024cda103d3d40ba232379c0c1853e8e181b58a65c4e954456cdcc040",
+        expand=False,
+    )
```

Comment:
Can you download the source tarball instead?

Reply:
done!